packages/coinstac-ui/app/main/index.js

```diff
@@ -131,7 +131,10 @@ loadConfig()
         core.unlinkFiles(run.id);
         mainWindow.webContents.send('local-run-error', {
           consName: consortium.name,
-          run: Object.assign(run, { error, endDate: Date.now() }),
+          run: Object.assign(
```

Comment:
this was causing an RPC issue where the error wasnt being sent, I'm guessing there's an issue rpc'ing the whole Error proto stuff